src/styles/blocks/about.scss

```diff
@@ -0,0 +1,43 @@
+.about {
+  &__content {
+    @include onTablet {
+      display: grid;
+      grid-template-columns: repeat(6, 1fr);
+      column-gap: 24px;
+    }
+
+    @include onDesktop {
+      grid-template-columns: repeat(12, 1fr);
+      column-gap: 24px;
```

Comment:
You have the same gap as onTablet, as onDesktop, I think you can use it just one time